lib/spack/spack/cmd/unload.py

```diff
@@ -4,20 +4,59 @@
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
 import argparse
-from spack.cmd.common import print_module_placeholder_help
+import sys
 
-description = "remove package from environment using `module unload`"
-section = "modules"
+import llnl.util.tty as tty
+
+import spack.cmd
+import spack.environment as ev
+import spack.util.environment
+import spack.user_environment as uenv
+
+description = "remove package from the user environment variables`"
```

Comment:
Are you sure? I added it to differentiate from Spack environments, which could be "the user's active environment"

Reply:
I guess I don't like the "environment variables" part since you would normally say "the user's environment".  But we have an overload with Spack environments.  @adamjstewart: any suggestions for how to disambiguate this?